src/lua/utils.h

```diff
@@ -577,6 +579,93 @@ void luaL_iterator_delete(struct luaL_iterator *it);
 int
 tarantool_lua_utils_init(struct lua_State *L);
 
+/** Same as luaL_checkstring but raises box error. **/
+static inline const char *
+luaT_checkstring(struct lua_State *L, int index)
+{
+	const char *name = lua_tostring(L, index);
+	if (name == NULL) {
+		diag_set(IllegalParams, "argument %d is not a string", index);
```

Comment:
1. Please make these functions `extern`.
2. You use different format of error messages for `luaT_checkint64` and these functions. Please unify them.

Reply:
Done.